CHANGELOG.md

```diff
@@ -184,6 +238,43 @@ your tests.
 
 [#7835]: https://github.com/googleapis/google-cloud-cpp/issues/7835
 
+**OTHER CHANGES**:
+
+* fix(storage): missing options in UpdateObject() ([#8193](https://github.com/googleapis/google-cloud-cpp/pull/8193))
+* feat(GCS+gRPC): implement CopyObject() ([#8190](https://github.com/googleapis/google-cloud-cpp/pull/8190))
```

Comment:
I am uncertain on the `GCS+gRPC` changes, that component is "experimental".  I think it is cleaner to omit them and then make an announcement when we go to private and/or public preview.  Thoughts?

Reply:
We have included them in the past. I am fine with not including them. Alternatively we can collect the gRPC changes in an h4 under Storage, after "OTHER CHANGES". Something like this:

#### (Experimental) Storage gRPC

* feat(GCS+gRPC): blah
* feat(GCS+gRPC): blah blah
* feat(GCS+gRPC): blah blah blah

Reply:
I think I prefer to not include them, but whatever is easier for you also works for me.